src/mscorlib/src/System/Runtime/Intrinsics/X86/Avx.PlatformNotSupported.cs

```diff
@@ -17,991 +17,1242 @@ public static class Avx
 
         /// <summary>
         /// __m256 _mm256_and_ps (__m256 a, __m256 b)
+        ///   VANDPS ymm, ymm, ymm/m256
         /// </summary>
         public static Vector256<float> And(Vector256<float> left, Vector256<float> right) { throw new PlatformNotSupportedException(); }
         /// <summary>
         /// __m256d _mm256_and_pd (__m256d a, __m256d b)
+        ///   VANDPD ymm, ymm, ymm/m256
         /// </summary>
         public static Vector256<double> And(Vector256<double> left, Vector256<double> right) { throw new PlatformNotSupportedException(); }
 
         /// <summary>
         /// __m256 _mm256_andnot_ps (__m256 a, __m256 b)
+        ///   VANDNPS ymm, ymm, ymm/m256
         /// </summary>
         public static Vector256<float> AndNot(Vector256<float> left, Vector256<float> right) { throw new PlatformNotSupportedException(); }
         /// <summary>
         /// __m256d _mm256_andnot_pd (__m256d a, __m256d b)
+        ///   VANDNPD ymm, ymm, ymm/m256
         /// </summary>
         public static Vector256<double> AndNot(Vector256<double> left, Vector256<double> right) { throw new PlatformNotSupportedException(); }
 
         /// <summary>
         /// __m256 _mm256_blend_ps (__m256 a, __m256 b, const int imm8)
+        ///   VBLENDPS ymm, ymm, ymm/m256, imm8
         /// </summary>
         public static Vector256<float> Blend(Vector256<float> left, Vector256<float> right, byte control) { throw new PlatformNotSupportedException(); }
         /// <summary>
         /// __m256d _mm256_blend_pd (__m256d a, __m256d b, const int imm8)
+        ///   VBLENDPD ymm, ymm, ymm/m256, imm8
         /// </summary>
         public static Vector256<double> Blend(Vector256<double> left, Vector256<double> right, byte control) { throw new PlatformNotSupportedException(); }
-                
+
         /// <summary>
         /// __m256 _mm256_blendv_ps (__m256 a, __m256 b, __m256 mask)
+        ///   VBLENDVPS ymm, ymm, ymm/m256, ymm
         /// </summary>
         public static Vector256<float> BlendVariable(Vector256<float> left, Vector256<float> right, Vector256<float> mask) { throw new PlatformNotSupportedException(); }
         /// <summary>
         /// __m256d _mm256_blendv_pd (__m256d a, __m256d b, __m256d mask)
+        ///   VBLENDVPD ymm, ymm, ymm/m256, ymm
         /// </summary>
         public static Vector256<double> BlendVariable(Vector256<double> left, Vector256<double> right, Vector256<double> mask) { throw new PlatformNotSupportedException(); }
 
         /// <summary>
         /// __m128 _mm_broadcast_ss (float const * mem_addr)
+        ///   VBROADCASTSS xmm, m32
         /// </summary>
-        public static Vector128<float> BroadcastElementToVector128(ref float source) { throw new PlatformNotSupportedException(); }
+        public static unsafe Vector128<float> BroadcastElementToVector128(float* source) { throw new PlatformNotSupportedException(); }
```

Comment:
@fiigii, should these be called `BroadcastScalarToVector128`?

Reply:
(for consistency with the other operations that take a scalar.

Reply:
IMO in this case we have prior art in the form of established Intel docs which use term `element` and programmers IMHO probably got used to read those instructions in this way - at least this is my experience

Reply:
That would be more consistent, I think

Reply:
Will update.

Reply:
Agree.

Reply:
And Avx2 counterparts should be updated as well.